### ISSUE TITLE
[C-2166] Inconsistent spacing in error messages for create activity fields

### ISSUE THREAD

**Issue by skwowet** [CrowdDotDev/crowd.dev#1382]
Currently, the display of spacing appears rather disjointed. It could be improved with better alignment of the error messages to ensure a cleaner look and improved readability.

<img width="491" alt="Screenshot 2023-08-27 at 4 04 09 PM" src="https://github.com/CrowdDotDev/crowd.dev/assets/20134207/94b845d6-d499-4e54-8ef8-36c79d0249c2">

<sub>From [SyncLinear.com](https://synclinear.com) | [C-2166](https://linear.app/crowddotdev/issue/C-2166/inconsistent-spacing-in-error-messages-for-create-activity-fields)</sub>

**Comment by Nike682631** [comment]
Can I take up this issue?

**Comment by yeganathan18** [comment]
@Nike682631 Thanks for your interest in contributing! assigned the issue to you 🙂

**Comment by Nike682631** [comment]
Yes I do need help. I was finding some issues with docker images trying to run the application. I'll post them soon here for your reference or in your discord server.

**Comment by peoray** [comment]
@Nike682631 I'd suggest you use Orbstack as a docker replacement but I'm not sure if that might help in your situation. Regardless, the installation process definitely needs a lot of improvement

**Comment by peoray** [comment]
picking this up :)
